Added a human interface to browse the repository

Daniel-KM opened the pull request:
Hi,

I added an optional human interface to browse the repository.

Sincerely,

Daniel Berthereau
Infodoc & Knowledge management

zerocrates commented:
I've cherry-picked the "default human error message" commit. It may take me a little longer to check out the "human interface" one.

Daniel-KM commented:
Hi,

I included your remarks and rebased the branch on the last commit.

I added some fixes and improvements too to cover the whole OAI-PMH standard. Even if some points are not useful currently, they may be later.

Sincerely,

Daniel Berthereau
Infodoc & Knowledge management

zerocrates commented:
I see a new commit that's rebased on my master, but it looks like at least some of the things I had issues with are still there (e.g., the hotlinked Omeka logo, the page title/header, the homepage).

Daniel-KM commented:
Hi,

I rebased all of them.

Sincerely,

Daniel Berthereau
Infodoc & Knowledge management

zerocrates commented:
Okay, I'm seeing them correctly now.
